scripts/tcs_startup.sh

```diff
@@ -19,8 +19,15 @@ listener="${TCF_HOME}/examples/common/python/connectors/direct/tcs_listener/tcs_
 # Read Listener port from config file
 listener_port=`grep listener_port ${TCF_HOME}/config/tcs_config.toml | awk {'print $3'}`
 
+lmdb_server="${TCF_HOME}/examples/common/python/shared_kv/remote_lmdb/lmdb_listener.py"
+
 start_tcs_components()
 {
+    echo "start the lmdb server"
+    python3 ${lmdb_server} &
+    sleep 3s
+    echo "lmdb server started"
```

Comment:
Can we create lmdb remote server functionality if lmdb remote_url is set in config file ? or we can use a argument to startup script ?

Reply:
The startup of lmdb remote server has been handled in the `lmdb_listener.py`. If `remote_url` is set in config file, this server would simply quit with warning and the local LMDB would be employed.

Reply:
implement as 3fa74144d8679cf83ec5694355266d3b7650b666, which would employ the remote LMDB if `remote_url` exists in config/tcs_config.toml